### ISSUE TITLE
Explanation on "How to claim these accounts"

### ISSUE THREAD

**Issue by dskbrd** [terror/usernames#3]
First of all thank you for the effort of creating this tool and the list published here.

My Question is:
How would you go forward to try and claim one of these inactive account names?
Is there some kind of workflow from Github for this usecase or do you have to contact the support directly?

**Comment by terror** [comment]
Your welcome!

You would need to contact GitHub support directly and let them know the
account is inactive. If the
account is truly inactive they will free it for you.

Many of these accounts may still be technically "active", so let me know if
you try to claim one
and find out it's active or you manage to snag an inactive one, so I can
remove them from
the list.

**Comment by terror** [comment]
Your welcome! Yes you are right, you would need to contact support.

If the account is truly inactive (there is a possibility that it isn't) they will free it for you.

**Comment by bloatoo** [comment]
How would you actually contact them? Their system is confusing

**Comment by terror** [comment]
> How would you actually contact them? Their system is confusing

Hi @bloatoo!

They have a contact form -> https://support.github.com/contact

**Comment by bloatoo** [comment]
What is it that I should do there? Trying to send them a request in the ways that I've figured has always been a dead end. There's also no straight-up way of sending them an email or something

**Comment by terror** [comment]
> What is it that I should do there? Trying to send them a request in the ways that I've figured has always been a dead end. There's also no straight-up way of sending them an email or something

Your best bet is that form -- if the account is truly inactive and the request is pure, you'll have no problem.

**Comment by bloatoo** [comment]
Got you. Which one of the topics should I choose? None of them include username squatting, and picking "Report abuse or spam" should be done from the user's profile. And even if I was to do it from the user's profile, it wouldn't give me a chance to send the request

**Comment by terror** [comment]
> Got you. Which one of the topics should I choose? None of them include username squatting, and picking "Report abuse or spam" should be done from the user's profile. And even if I was to do it from the user's profile, it wouldn't give me a chance to send the request

I don't see any topics to be chosen from on the form? All you need is a simple subject and body detailing your request.

**Comment by bloatoo** [comment]
![image](https://user-images.githubusercontent.com/71422507/116735423-27855d80-a9f7-11eb-8b73-68cda53725bd.png)

The page looks like this, what should I do from here?

**Comment by terror** [comment]
> The page looks like this, what should I do from here?

Is that what you get when you visit https://support.github.com/contact ?

**Comment by bloatoo** [comment]
Yup.

**Comment by terror** [comment]
> Yup.

I would just go with account or billing issues. They will do their best to handle the request regardless of the topic I assume.

**Comment by bloatoo** [comment]
Got you, thanks

**Comment by ghost** [comment]
Apparently, GitHub has updated its username policy and is no longer accepting requests to release dormant usernames.
![image](https://user-images.githubusercontent.com/65695924/123904341-f17c3f80-d98d-11eb-9664-c4e4263b43f7.png)

**Comment by poundsigntown** [comment]
Yeah... they even added a "category" in the contact form dropdown, it seems, for no other reason than _just_ to trigger this dialog when its selected:

![image](https://github.com/terror/usernames/assets/167234583/6bd2e45b-8a57-4186-a1b8-b09b159e653b)
